src/Blocks/components/checkboxes/checkboxes.php

```diff
@@ -10,13 +10,33 @@
 
 $manifest = Components::getManifest(__DIR__);
 
+$checkboxesId = Components::checkAttr('checkboxesId', $attributes, $manifest);
 $checkboxesContent = Components::checkAttr('checkboxesContent', $attributes, $manifest);
+$checkboxesName = Components::checkAttr('checkboxesName', $attributes, $manifest);
+
+// Add internal counter name key.
+$checkboxesContent = (string) preg_replace_callback('/name=""/', function () use ($checkboxesName) {
+	return 'name="' . $checkboxesName . '"';
+}, $checkboxesContent);
+
+// Add internal counter id key.
+$indexId = 0;
+$checkboxesContent = (string) preg_replace_callback('/id=""/', function () use (&$indexId, $checkboxesId) {
+	return 'id="' . $checkboxesId . '[' . $indexId++ . ']"';
+}, $checkboxesContent);
```

Comment:
This will loop the `$checkboxesContent` in the preg replace? Never used this, neat! 👍🏼